src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -290,7 +265,7 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                     try
                     {
                         fileTraceWriter.Info(text);
-                        traceWriter.Info(text);
+                        invocationContext.TraceWriter.Info(text);
```

Comment:
For Node, TraceWriter will always be there. But C# (for example) may not have it -- only if the user's function requests it as a parameter.

---
In reply to: [77687749](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r77687749) [](ancestors = 77687749)